### PR TITLE
keepalived: Update keepalived to 1.3.5 and use client-go

### PR DESCRIPTION
**Pull request by aledbf**
replaces #2041

### CODE REVIEW

**Comment by bowei** [comment]
I see "do not merge" in the commit comment. Should I review this?

**Comment by jonaz** [comment]
Whats the status of this? We are thinking about using this in production but want the latest keepalived and multiple VIPS.

**Comment by aledbf** [comment]
@bowei this is ready to be merged

File: keepalived-vip/keepalived.tmpl [hunk 1]
```diff
@@ -5,10 +5,15 @@ global_defs {
   vrrp_iptables {{ .iptablesChain }}
 }
 
+vrrp_script chk_haproxy {
```
Comment (bowei) [inline keepalived-vip/keepalived.tmpl]: should this be templated out if ha proxy is not used?

Comment (bowei) [inline keepalived-vip/keepalived.tmpl]: (is HAProxy an optional thing?)

Comment (bowei) [inline keepalived-vip/keepalived.tmpl]: otherwise looks ok

**Comment by maikotz** [comment]
Any update on this ?

**Comment by dseeley-sky** [comment]
For anyone else who needs this urgently, I've merged master into this pull request on a forked repo (https://github.com/dseeley-sky/contrib/tree/2608/keepalived-vip), and uploaded the resultant build to dockerhub: skyvtp/kube-keepalived-vip:0.10.2608

**Comment by dseeley-sky** [comment]
@bowei - I moved the rebased code to my personal GitHub account (https://github.com/dseeley/contrib/tree/2608/keepalived-vip) as this one doesn't have CLA approval (and won't get it quickly), and submitted #2733 . The other changes have restored functionality that was lost, addressed your comment above ("should this be templated out if ha proxy is not used?"), and upgraded to v1.3.6.  This image is on dockerhub: skyvtp/kube-keepalived-vip:0.10.2608.4

**Pull request closed without merging**